### PR TITLE
Allow for config outside of vendor directory

### PR DESCRIPTION
**Pull request by timneutkens**
As we discussed in #39. If this solution is ok I will update readme.md so new users will use this instead of editing the vendor package.

### CODE REVIEW

File: helpers/config-loader.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 module.exports = function(file, plugins) {
-  if (plugins.globby.sync('./config/' + file).length) {
+  if (plugins.globby.sync('../../../dev/tools/frontools/configs/' + file).length) {
```
Comment (Igloczek) [inline helpers/config-loader.js]: Let's user here `projectPath` variable.

File: helpers/config-loader.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+    return require('../../../../dev/tools/frontools/configs/' + file);
+  } else if (plugins.globby.sync('./config/' + file).length) {
```
Comment (Igloczek) [inline helpers/config-loader.js]: I'm using Stroustrup indentation style.

File: helpers/config-loader.js (hunk 1 above)
Comment (timneutkens) [inline helpers/config-loader.js]: `this.opts.configs.projectPath` is not available inside of the helpers, that's why I didn't use it. What would your preferred solution to this be?

File: helpers/config-loader.js (hunk 2 above)
Comment (timneutkens) [inline helpers/config-loader.js]: Changed this.

File: helpers/config-loader.js (hunk 1 above)
Comment (Igloczek) [inline helpers/config-loader.js]: Pass them as function parameter.

Comment (timneutkens) [inline helpers/config-loader.js]: Updated

File: gulpfile.js [hunk 3]
```diff
@@ -24,7 +25,7 @@ require('gulp-task-loader')({
 });
 
 // define task for each theme, locale, lang, processing type etc.
-// gulp can't run same task in parallel, so we need different names
+// gulp can't run same task in parallel, so we need different namdes
```
Comment (Igloczek) [inline gulpfile.js]: Typo "names"

Comment (timneutkens) [inline gulpfile.js]: Fixed ;)

**Comment by Igloczek** [comment]
Thanks.
BTW. Can I ask you to create future PR to `develop` branch, it's much easier to maintain versioning (master should contain only releases)

**Comment by timneutkens** [comment]
Sure will do 👍

**Pull request merged**